Extensions/TextInput/textinputruntimeobject.ts

```diff
@@ -48,7 +48,9 @@ namespace gdjs {
   /**
    * Shows a text input on the screen the player can type text into.
    */
-  export class TextInputRuntimeObject extends gdjs.RuntimeObject {
+  export class TextInputRuntimeObject
+    extends gdjs.RuntimeObject
+    implements gdjs.OpacityHandler {
```

Comment:
I see the text input handles size (getWidth, setWidth, etc.) so you should be able to add ResizableHandler?
And maybe we should add the ScalableHandler too?

Reply:
Every objects has a dimension (but it can't necessarily be changed) so expressions are in BaseObject.
I'll add a comment.

Reply:
Indeed, I though it was the deprecated one.